src/renderer/entities/wallet/lib/account-utils.ts

```diff
@@ -121,19 +121,14 @@ function isAccountWithShards(accounts: AnyAccount | VaultShardAccount[]): accoun
   return Array.isArray(accounts) && isVaultShardAccount(accounts[0]);
 }
 
-function isChainDependant(account: AnyAccountDraft): boolean {
+function isChainDependant(account: AnyAccountDraft) {
```

Comment:
Maybe completely remove `isChainDependant` and use `accountService.isChainAccount` where needed?

Reply:
Yeah, I would have definitely done that